### PR TITLE
20230614 bumps

### PR DESCRIPTION
**Pull request by peterthomassen**
Takes care of several dependabot PRs.

### CODE REVIEW

File: www/webapp/package.json [hunk 1]
```diff
@@ -9,10 +9,10 @@
     "lint:fix": "vue-cli-service lint --fix"
   },
   "dependencies": {
-    "@fontsource/roboto": "^4.5.8",
+    "@fontsource/roboto": "^5.0.3",
     "@mdi/font": "^6.9.96",
     "@mdi/js": "~7.2.96",
-    "axios": "^0.27.2",
+    "axios": "^1.4.0",
```
Comment (Rotzbua) [inline www/webapp/package.json:15]: Will break

File: www/webapp/package.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
     "@vue/cli-service": "^5.0.8",
     "eslint": "^8.31.0",
     "eslint-plugin-vue": "^9.8.0",
-    "sass": "~1.32.13",
+    "sass": "~1.63.3",
```
Comment (Rotzbua) [inline www/webapp/package.json:34]: Adds the huge amount of deprecation notifications.

File: www/webapp/package.json [hunk 3]
```diff
@@ -9,10 +9,10 @@
     "lint:fix": "vue-cli-service lint --fix"
   },
   "dependencies": {
-    "@fontsource/roboto": "^4.5.8",
+    "@fontsource/roboto": "^5.0.3",
```
Comment (Rotzbua) [inline www/webapp/package.json:12]: Tested. Seems ok.

**Review by Rotzbua — CHANGES_REQUESTED** [review]
@peterthomassen You can silence dependabot with commands in the PR like `@dependabot ignore this minor version`

File: www/webapp/package.json (hunk 2 above)
Comment (peterthomassen) [reply www/webapp/package.json:34]: Is that a problem? -- Perhaps it makes more sense to first finish #699? The new Vue version hopefully has code that doesn't use much of the deprecated stuff.

File: www/webapp/package.json (hunk 1 above)
Comment (Rotzbua) [reply www/webapp/package.json:15]: Looks good.

File: www/webapp/package.json (hunk 2 above)
Comment (Rotzbua) [reply www/webapp/package.json:34]: The deprecation messages are over hundreds and would hide legitimate warnings.
I would recommend to bump it with the migration to Vue3.

File: www/webapp/src/router/index.js [hunk 4]
```diff
@@ -158,7 +158,7 @@ router.beforeEach((to, from, next) => {
   const user = useUserStore();
   if (sessionStorage.getItem('token') && !user.authenticated) {
     const token = JSON.parse(sessionStorage.getItem('token'));
-    HTTP.defaults.headers.common['Authorization'] = 'Token ' + token.token;
+    HTTP.defaults.headers['Authorization'] = 'Token ' + token.token;
```
Comment (Rotzbua) [inline www/webapp/src/router/index.js:161]: For style, the other code snippets are:
`HTTP.defaults.headers.Authorization`

**Review by Rotzbua — CHANGES_REQUESTED** [review]

File: www/webapp/src/router/index.js (hunk 4 above)
Comment (peterthomassen) [reply www/webapp/src/router/index.js]: done

File: www/webapp/package.json (hunk 2 above)
Comment (peterthomassen) [reply www/webapp/package.json]: sounds good, removed

**Comment by peterthomassen** [comment]
> For documentation there is no official `axios` migration guide, but there is 3rd party guide: https://github.com/bmuenzenmeyer/axios-1.0.0-migration-guide/

yeah, that's what I went over.

**Comment by peterthomassen** [comment]
I saw that I missed #704. I'll check, and if it seems compatible, I'll add it to this PR.

**Review by nils-wisiol — APPROVED** [review]
lgtm

**Pull request merged**